test/fzf.vader

```diff
@@ -147,6 +147,23 @@ Execute (fzf#wrap):
   let opts = fzf#wrap({})
   Assert opts.options =~ '^--color=fg:'
 
+Execute (fzf#shellesc with sh):
+  AssertEqual '""', fzf#shellesc('', 'sh')
+  AssertEqual '"\"\""', fzf#shellesc('""', 'sh')
+  AssertEqual '"foobar>"', fzf#shellesc('foobar>', 'sh')
+  AssertEqual '"\\""', fzf#shellesc('\"', 'sh')
```

Comment:
The first `escape` in `s:execute` to handle `%#!` breaks `FZF` because fzf is run in the batchfile.
`%` are escaped as `%%` in batchfiles and `^%` in the interactive shell.